ui/src/info_tag.test.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { render } from '@testing-library/react'
+import React from 'react'
+import { XInfoTag, InfoTag } from './info_tag'
+
+let infoTagProps: InfoTag
+
+describe('InfoTag.tsx', () => {
+  beforeEach(() => {
+    infoTagProps = {
+      name: 'info-tag',
+      color: 'black',
+      label: '1',
+    }
+  })
+
+  it('Renders data-test attr', () => {
+    const { queryByTestId } = render(<XInfoTag model={infoTagProps} />)
+    expect(queryByTestId(infoTagProps.name)).toBeInTheDocument()
+  })
+
+  it('Renders label', () => {
```

Comment:
I believe that visual regression is related to how it looks, making sure that changes to the logic don't impact the colors and other properties of our components. In this case, label is a requirement for our component and in this test I was checking if the value passed to the prop is properly rendered. I removed the test since we don't have it for other components.